fixitfriday.ui/src/components/Survey/SurveyVisivilityAdministration/SurveyQuestions.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import Col from 'react-bootstrap/Col';
+import { Row } from 'react-bootstrap';
+import { SurveyQuestionsPropsType } from './types/SurveyQuestionsPropsType';
+import { SurveyQuestionType } from '../types/SurveyQuestionType';
+import Question from './Question';
+
+const SurveyQuestions: FC<SurveyQuestionsPropsType> = (props: SurveyQuestionsPropsType) => {
+  const { survey, disabled } = props;
+
+  const questions =
+    survey && survey.length > 0 ? (
+      survey.map((s: SurveyQuestionType) => (
```

Comment:
```suggestion
      survey.map(({ id, question, disabled }: SurveyQuestionType) => (
```

Same here, so we have shorter names.